mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -1487,27 +1489,32 @@ msgid "hellothere"
 msgstr "???: Hello there!"
 
 msgid "kmere"
-msgstr "Why don't you come over here and talk to me?"
+msgstr "Why don't you come over here and talk to me?" # Why does Aeble wish to talk to the player? 
+# The dialogue that follows is great, but maybe there’s a way to make it feel less obviously like exposition,
```

Comment:
Please make sure these comments are removed and discussions opened in another format: forums/github issues/or discord.